### PR TITLE
Fix the PAL & Space Bubble interaction

### PR DESCRIPTION
**Pull request by zfox23**
[Trello Card](https://trello.com/c/v3Abvhlu/43-bug-when-pal-open-bubble-doesn-t-work-correctly)

Test Plan:
* In a domain with 2+ avatars present, perform a space bubble sanity check without the PAL open to see if it works as expected.
* In the same domain, as the admin, enable the PAL and ensure you can see and hear all avatars (even if they've ignored you).
* In the same domain, as the admin, enable the space bubble and the PAL, then walk into another avatar. That avatar should disappear, but their sphere overlay should remain. You shouldn't be able to hear them.
* In the same domain, as the non-admin, enable the space bubble and the PAL, then walk into another avatar. That avatar should disappear, *and* their sphere overlay should disappear. You shouldn't be able to hear them.
* In the same domain, as the non-admin, mute the admin. The admin should not be able to hear the non-admin, and vice versa. As the admin, open the PAL. The admin and non-admin should now be able to speak to and hear each other.

More detailed test plan pending a write-up from @howard-stearns.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 495925a017b0191c6723c8ff6329bf2270909fb0 available at: https://gist.github.com/46800b7931e3b5656111e4c7d2de2762<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-495925a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-495925a.dmg

File: libraries/networking/src/udt/PacketHeaders.h [hunk 1]
```diff
@@ -105,7 +105,8 @@ class PacketTypeEnum {
         UsernameFromIDReply,
         ViewFrustum,
         RequestsDomainListData,
-        LAST_PACKET_TYPE = RequestsDomainListData
+        ExitingSpaceBubble,
+        LAST_PACKET_TYPE = ExitingSpaceBubble //FIXME
```
Comment (howard-stearns) [inline libraries/networking/src/udt/PacketHeaders.h]: fixme?

Comment (zfox23) [inline libraries/networking/src/udt/PacketHeaders.h]: Embarrassing, thanks. I fixed the thing but didn't remove the comment.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b7d64804c1397bd158460915618886a285d7de70 available at: https://gist.github.com/46800b7931e3b5656111e4c7d2de2762<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-b7d6480.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-b7d6480.dmg

**Comment by davidkelly** [comment]
When I'm the admin, and a non-admin ignores or mutes me, I can still see them.  But, I cannot hear them.  My understanding from the test plan is the ignored/muted person should still both see and hear the ignorer/muter.  Is that correct?  If so, this is a QA Fail.

Potentially related:  At one point, after some ignore/mute on the non-admin side, my PAL had the ignorer listed twice (same uuid, same session display name).  Unsure how to repro yet, or if it is related to this PR in particular.  I'll work on repro case for this.

**Comment by Nex-Pro** [comment]
We don't have ignore icon showing for the non-admins running this build.

**Comment by zfox23** [comment]
@Nex-Pro can you please attach a screenshot showing what you mean?

**Comment by zfox23** [comment]
Hoo boy, this is a QA fail. After examining some corner cases, I don't think I've fixed this issue. Back to the drawing board...

**Comment by hifi-gustavo** [comment]
Build console output for commit: dec41f67cc53d6c8b2a6dd58220198179d42dbba available at: https://gist.github.com/46800b7931e3b5656111e4c7d2de2762<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-dec41f6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-dec41f6.dmg

**Comment by zfox23** [comment]
[BUG - When PAL open bubble doesn't work correctly](https://trello.com/c/v3Abvhlu/43-bug-when-pal-open-bubble-doesn-t-work-correctly)

**Comment by hifi-gustavo** [comment]
Build console output for commit: afa6d54ac469a66ff8d8ae52c8b3456c391b47fa available at: https://gist.github.com/46800b7931e3b5656111e4c7d2de2762<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-afa6d54.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-afa6d54.dmg

**Comment by davidkelly** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: afa6d54ac469a66ff8d8ae52c8b3456c391b47fa available at: https://gist.github.com/46800b7931e3b5656111e4c7d2de2762<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-afa6d54.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9323-afa6d54.dmg

**Pull request merged**